tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -283,28 +285,26 @@ func (rb *PSATestSuite) TestPSA() {
 		if role == roleCustomCreateNS {
 			customRole = true
 		}
-		rb.Run("Add PSA labels on the namespaces created by admins ", func() {
-			createProjectAsAdmin, err := createProject(rb.client, rb.cluster.ID)
-			rb.adminProject = createProjectAsAdmin
-			require.NoError(rb.T(), err)
+		createProjectAsAdmin, err := createProject(rb.client, rb.cluster.ID)
```

Comment:
the rest of this for loop breaks everything up into its own rb.run. Is there a reason this specific subsection/run was removed?

Reply:
Because it was rewriting the test over and over again except with a number appended to the end of it. This is really as setup step not its own test, and for qase reporting reasons this made no sense. I asked @anupama2501 about it, but she can weigh in if we want it handled another way.